packages/neos-ui/src/Containers/Modals/SelectNodeType/nodeTypeItem.js

```diff
@@ -31,8 +31,10 @@ class NodeTypeItem extends PureComponent {
 
     render() {
         const {ui, name} = this.props.nodeType;
-        const icon = $get('icon', ui);
         const label = $get('label', ui);
+        const useLargeIcon = ('largeIcon' in ui);
```

Comment:
what's the difference between `('largeIcon' in ui);` and $get('icon', ui). Why not use the same method?

Reply:
This checks if we have the `largeIcon` property in the options. 
And then we use the `icon` or the `largeIcon`